### PR TITLE
Flz 8667 live event analytics attendance

### PR DESCRIPTION
**Pull request by adi-heffer-folloze**

### CODE REVIEW

**Comment by TanyaFolloze** [comment]
Task linked: [FLZ-8667 Show event attendance on pulse](https://app.clickup.com/t/3749831/FLZ-8667)

File: src/analytics/LiveEventAnalytics.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default class LiveEventAnalytics {
+    private fetchService: FetchService;
+
+    constructor(fetcher: FetchService) {
+        this.fetchService = fetcher;
+    }
+
+    trackAttendance(
+        trackType: TrackAttendanceType,
+        data: {
```
Comment (TanyaFolloze) [inline src/analytics/LiveEventAnalytics.ts:21]: create a type for this

File: tests/analytics/live-event-analytics.test.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                params.activityType = "viewed_recording";
+                await liveEventAnalytics.trackAttendance("create", params);
+
+                expect(postSpy).toHaveBeenCalledWith(expect.any(String), expect.objectContaining({activity_type: 2}));
+            });
+        });
+    });
+
+    describe("In preview", () => {
+            beforeEach(async () => {
```
Comment (TanyaFolloze) [inline tests/analytics/live-event-analytics.test.ts:91]: something weird with indentation

**Review by TanyaFolloze — APPROVED** [review]

File: tests/analytics/live-event-analytics.test.ts (hunk 2 above)
Comment (adi-heffer-folloze) [reply tests/analytics/live-event-analytics.test.ts:91]: It's 4 spaces, which is out standard. I tried formatting with both lint and prettier, and they didn't change anything about the indentation

**Pull request merged**